main.go

```diff
@@ -39,14 +56,16 @@ func main() {
 		log.Fatalf("Error setting MySQL logger: %v", err)
 	}
 
-	// Check if "version" arg is passed.
-	argsWithProg := os.Args
+	args := flag.Args()
 
-	if len(argsWithProg) > 1 {
-		switch argsWithProg[1] {
-		case "version":
-			fmt.Println("LazySQL version: ", version)
-			os.Exit(0)
+	if len(args) > 1 {
+		fmt.Fprintln(os.Stderr, "Only a single connection is allowed")
+		os.Exit(1)
+	} else if len(args) == 1 {
+		err := components.InitFromArg(args[0])
+		if err != nil {
+			fmt.Fprintln(os.Stderr, err)
+			os.Exit(1)
```

Comment:
log.Fatal is already used on errors, so you should reuse it

```suggestion
			log.Fatal(err)
```

log.Fatal print errors to stderr too, then call os.Exit(1)